webapp/tests/karma/ts/services/enketo.service.spec.ts

```diff
@@ -127,6 +127,7 @@ describe('Enketo service', () => {
         },
         { provide: ZScoreService, useValue: { getScoreUtil: sinon.stub().resolves(sinon.stub())} },
         { provide: TransitionsService, useValue: transitionsService },
+        { provide: TranslateService, useValue: { get: sinon.stub().callsFake((key) => `translated key ${key}`) } },
```

Comment:
Can you please just stub the function here but make the callsFake inside the test case? That helps the unit test to control its setup and easier to understand when reading it. Other unit test might need to defined different behaviour for this stub, for example making it throw an exception.